src/components/Incorporation/ShareStructure.vue

```diff
@@ -172,11 +172,11 @@ export default class ShareStructure extends Mixins(CurrencyLookupMixin) {
     shareStructureForm: FormIF
   }
 
-  @Prop() readonly initialValue!: ShareClassIF
-  @Prop() readonly activeIndex: number
-  @Prop() readonly parentIndex: number
-  @Prop() readonly shareId: string
-  @Prop() readonly shareClasses: ShareClassIF[]
+  @Prop({ default: null }) readonly initialValue!: ShareClassIF
+  @Prop({ required: true }) readonly activeIndex!: number
+  @Prop({ required: true }) readonly parentIndex!: number
+  @Prop({ required: true }) readonly shareId!: string
+  @Prop({ required: true }) readonly shareClasses!: ShareClassIF[]
```

Comment:
All these are required (and I have confirmed in the calling component that all have an initial value).